integrations/amazon_bedrock/tests/test_chat_generator_utils.py

```diff
@@ -676,25 +682,18 @@ def test_callback(chunk: StreamingChunk):
                 meta={
                     "model": model,
                     "index": 0,
-                    "finish_reason": "tool_use",
+                    "finish_reason": "tool_calls",
                     "usage": {"prompt_tokens": 364, "completion_tokens": 71, "total_tokens": 435},
                     "reasoning_contents": [],
                 },
             )
         ]
 
+        # TODO Could update to check all streaming chunks
         # Verify streaming chunks were received for all content
         assert len(streaming_chunks) == 21
         assert streaming_chunks[1].content == "Certainly! I can"
         assert streaming_chunks[2].content == " help you find out"
-        assert streaming_chunks[12].meta["tool_calls"] == [
-            {
-                "index": 1,
-                "id": "tooluse_pLGRAmK7TNKoZQ_rntVN_Q",
-                "function": {"arguments": "", "name": "weather_tool"},
-                "type": "function",
-            }
-        ]
```

Comment:
I would still check that the tool call is present. No?

Reply:
Resolved by checking all streaming chunks